src/Code.gs

```diff
@@ -830,7 +862,23 @@ const ChatGPTApp = (function () {
     }
   }
 
-  function runOpenAIAssistant(assistantId, prompt) {
+  function getFileType(fileId) {
+    var file = DriveApp.getFileById(fileId);
+    var mimeType = file.getMimeType();
+    
+    switch (mimeType) {
+      case 'application/vnd.google-apps.spreadsheet':
+        return 'spreadsheet';
+      case 'application/vnd.google-apps.document':
+        return 'document';
+      case 'application/vnd.google-apps.presentation':
+        return 'presentation';
+      default:
+        throw new Error('Unsupported file type: ' + mimeType);
```

Comment:
Utilise plutôt les `` pour pouvoir utiliser des variables (${}) pour plus d'homogénéité.